main.nf

```diff
@@ -480,13 +489,16 @@ if (params.target_bed)          summary['Target BED']        = params.target_bed
 if (step)                       summary['Step']              = step
 if (params.tools)               summary['Tools']             = tools.join(', ')
 if (params.skip_qc)             summary['QC tools skip']     = skipQC.join(', ')
+if (params.trimFastq)           summary['Fastq trim']        = "Fastq trim selected"
 
 if (params.no_intervals && step != 'annotate') summary['Intervals']         = 'Do not use'
 if ('haplotypecaller' in tools)                summary['GVCF']              = params.no_gvcf ? 'No' : 'Yes'
 if ('strelka' in tools && 'manta' in tools )   summary['Strelka BP']        = params.no_strelka_bp ? 'No' : 'Yes'
 if (params.sequencing_center)                  summary['Sequenced by']      = params.sequencing_center
 if (params.pon && 'mutect2' in tools)          summary['Panel of normals']  = params.pon
 
+summary['Trim Fastq']      = params.trimFastq ? 'Yes' : 'No'
+//summary['Saved Trimmed Fastq']        = params.saveTrimmed ? 'Yes' : 'No'
```

Comment:
You let that line commented, any reason?

Reply:
no reason, missed that one.